packages/tailwind-config/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@documenso/tsconfig/base.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "noEmit": true,
+  },
+  "include": ["**/*.cjs", "**/*.js"],
+  "exclude": ["dist", "build", "node_modules"]
```

Comment:
If we are extending from the tsconfig package, do we need to add the same excluded directories? 😶

Reply:
Not strictly required but preferred since we’re changing up the include we don’t want to confuse devs as to what’s being excluded

Reply:
This file and this, https://github.com/documenso/documenso/blob/ca40e983e3f35e8232477cb8e85061fcc35eff98/packages/eslint-config/tsconfig.json seems to be the exact same, so do we need to add the same complierOptions, include and exclude  keys?

Reply:
I believe that's what I did, since I copy pasted the file 🙈